sale_order_line_description/models/sale.py

```diff
@@ -30,12 +30,12 @@ def product_id_change(
             self, pricelist, product, qty=0, uom=False, qty_uos=0,
             uos=False, name='', partner_id=False, lang=False,
             update_tax=True, date_order=False, packaging=False,
-            fiscal_position=False, flag=False
+            fiscal_position=False, flag=False, context=None
     ):
         res = super(SaleOrderLine, self).product_id_change(
             pricelist, product, qty, uom, qty_uos, uos, name,
             partner_id, lang, update_tax, date_order, packaging,
-            fiscal_position, flag)
+            fiscal_position, flag, context)
```

Comment:
Take the occasion to name each keyword argument in case any inheritance change the argument order